runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -14665,3 +14665,18 @@ J9::Z::TreeEvaluator::inlineIntegerStringSize(TR::Node* node, TR::CodeGenerator*
 
    return node->setRegister(lengthReg);
    }
+
+TR::Register*
+J9::Z::TreeEvaluator::inlineOnSpinWait(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   TR::Instruction* cursor = new (cg->trHeapMemory()) TR::S390NOPInstruction(TR::InstOpCode::NOP, 2, node, cg);
+
+   TR::Compilation *comp = cg->comp();
+   static const bool printIt = feGetEnv("TR_showPauseOnSpinWait") != NULL;
```

Comment:
TR_showPauseOnSpinWait would be a little misleading since "PAUSE" instruction is x86 specific but I have to use the same env variable because it was used in shared code and x86!